requirements-dev.txt

```diff
@@ -8,7 +8,7 @@ nose_parameterized==0.3.3
 profanityfilter
 lxml==4.5.0
 splinter>=0.13.0
--e git+https://github.com/qld-gov-au/ckanext-data-qld@develop#egg=ckanext-data_qld
+-e git+https://github.com/salsadigitalauorg/ckanext-data-qld@feature/Enhancements_to_the_data_request_functionality#egg=ckanext-data_qld
```

Comment:
Once data-qld PR (https://github.com/qld-gov-au/ckanext-data-qld/pull/71) has been approved and merged into develop, I will update this to point to https://github.com/qld-gov-au/ckanext-data-qld@develop

Reply:
https://github.com/qld-gov-au/ckanext-data-qld/pull/71 has been merged (please make a note that we need to have this version and ckanext-data-qld (16 june 2020) tagged and released at the same time.